src/Components/Pokemons/Pokemon.js

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from "react";
+import "./Pokemon.css";
+import Card from "../Card/Card";
+
+const Pokemon = () => {
+  const [pokemonList, setPokemonList] = useState([]);
+
+  const getPokemonApi = async () => {
+    try {
+      const response = await fetch("https://pokeapi.co/api/v2/pokemon/");
+      const data = await response.json();
+      const pokemonQualities = [];
+
+      for (const pokemon of data.results) {
+        const pokemonUrl = await fetch(pokemon.url);
+        const pokemonDetails = await pokemonUrl.json();
+
+        const { name, sprites, weight, base_experience } = pokemonDetails;
+        pokemonQualities.push({
+          name: name,
```

Comment:
If the key and value has the same name, it can only appear once without breaking. For example, name